gql/resolver.go

```diff
@@ -601,8 +601,15 @@ func (dr *dealResolver) sealingState(ctx context.Context) string {
 		log.Warnw("error getting sealing status for sector", "sector", dr.SectorID, "error", err)
 		return "Sealer: Sealing"
 	}
-
-	return "Sealer: " + string(si.State)
+	if len(si.Deals) > 0 {
+		for _, d := range si.Deals {
+			if d == dr.ProviderDealState.ChainDealID {
+				return "Sealer: " + string(si.State)
+			}
+		}
+		return "Sealer: failed - deal not found in sector"
```

Comment:
I'd suggest we include the sector number in the error message here to make it easier for the the user to go and check on the sector status
```suggestion
		return fmt.Sprintf("Sealer: failed - deal not found in sector %d", si.SectorID)
```